lib/common/service.js

```diff
@@ -48,6 +50,8 @@ function Service(config, options) {
   this.authClient = this.makeAuthenticatedRequest.authClient;
   this.baseUrl = config.baseUrl;
   this.getCredentials = this.makeAuthenticatedRequest.getCredentials;
+  this.globalInterceptors = arrify(options.interceptors_);
```

Comment:
We need to be able to reference any later-assigned global interceptors, so a pointer to the original global interceptors object can't end up combined with this instance's ones. Likewise, we can't have something assigned from a Service instance's get muddied up with the global interceptors.

Did I say interceptors enough?